tests/container-scripts/setup-validators.sh

```diff
@@ -40,8 +40,9 @@ do
     GAIA_HOME="--home /validator$i"
     GENTX_HOME="--home-client /validator$i"
     ARGS="$GAIA_HOME --keyring-backend test"
+    KEY_ARGS="--algo secp256k1 --coin-type 118"
```

Comment:
The default keys are eth_secp256k1, and separately the default HD path is m/44'/60'/0'/0/0 like Ethereum, so in our testnet instructions we will need to tell users to specify both of these arguments to `althea keys add ...`